push_notifications/conf/legacy.py

```diff
@@ -14,6 +14,9 @@ class empty(object):
 
```

Comment:
Why is this added to the legacy config? If we're no longer defaulting to it / recommending it I don't think it's worth adding more features to it.

Reply:
But I think it should still work. It's still default.

It's another discussion, but FCM and APNS support for multiple applications (one shared token) and the app config (no sharing) make it seem like another approach would be better, from a code and configuration standpoint.

Reply:
I agree with @jleclanche here. We want to get people using the new format and drop legacy. Legacy was provided as a way to keep existing configurations working.

Reply:
But shouldn't that be a separate change? It's still default now. Shouldn't it at least be deprecated first, and then we remove support?

Reply:
I'm not saying remove legacy. I'm saying only add the JWT support to the new config format. I guess if the work is already done though we should just move forward with it as is.

Reply:
It was trivial to support legacy. If a smaller change was desired, supporting the *new* config would have been a separate change.